public/search.php

```diff
@@ -1,17 +1,44 @@
-<?php require_once('includes/header.php'); ?>
+<?php 
+require_once('includes/header.php');
+require_once('includes/db.php');
+
+$mysqli = db_connection();
+$state_id = $mysqli->real_escape_string($_POST['state']);
```

Comment:
It is advisable to use a GET action for search so that the user can save the search query and return to it later. Can we update the form to use GET instead of POST, please?